third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -173,6 +173,13 @@ struct LoadOpConversion : public ConvertOpToLLVMPattern<triton::LoadOp>,
       LLVM_DEBUG(llvm::dbgs() << " vec = " << vec << '\n');
     }
 
+    if (vec == 1 && numElems > 1) {
+      auto maskStr = !llMask ? "no mask" : std::to_string(getMaskAlignment(mask));
```

Comment:
It is probably better to have an integer for "no mask" case? Also can we add the value of vec prior to the "if (llMask)" statement in the remark?